caas/kubernetes/provider/k8s.go

```diff
@@ -254,14 +254,22 @@ func (k *kubernetesClient) EnsureOperator(appName, agentPath string, config *caa
 		}
 	}
 
+	storageTags := map[string]string{labelOperator: appName}
```

Comment:
Might be worthwhile adding this as the last, rather than the first, pair...I can see how it can be overwritten by either CharmStorage.ResourceTags here or config.ResourceTags below

Reply:
ok